lib/matplotlib/mlab.py

```diff
@@ -2287,7 +2287,7 @@ def log2(x, ln2=math.log(2.0)):
             return len(bin_n)
 
 
-@cbook.deprecated("2.2")
+@cbook.deprecated("2.2", 'numpy.mod(n, 2)')
```

Comment:
This isn't quite right. 

From: 
```python
import numpy
from matplotlib import mlab
N = [37, 48, 64]
print('numpy:\t', list(map(lambda x: numpy.mod(x, 2), N)))
print('mlab:\t', list(map(mlab.ispower2, N)))
```

I get:
```
numpy:	 [1, 0, 0]
mlab:	 [0, 0, 6]
```

Reply:
Oh yeah, that function doesn't quite do what it says on the tin (not that my alternative actually worked for determining if a number is a power of 2!)